# zrj: add missing link against -lz in libdcmdata.so (nothing better came to mind..)
# NOTYPE  GLOBAL DEFAULT  UND {deflate inflate etc}
# fixes graphics/aeskulap (binutils 2.27 ld.gold is unhappy)
--- dcmdata/libsrc/CMakeLists.txt.intermediate	2016-12-15 17:28:21.000000000 +0200
+++ dcmdata/libsrc/CMakeLists.txt
@@ -1,3 +1,7 @@
 # invoke macro to build and declare installation files
 SET(SOURCES cmdlnarg dcbytstr dcchrstr dccodec dcdatset dcddirif dcdicdir dcdicent dcdict dcdictzz dcdirrec dcelem dcerror dcfilefo dchashdi dcistrma dcistrmb dcistrmf dcistrmz dcitem dclist dcmetinf dcobject dcostrma dcostrmb dcostrmf dcostrmz dcpcache dcpixel dcpixseq dcpxitem dcrleccd dcrlecce dcrlecp dcrledrg dcrleerg dcrlerp dcsequen dcstack dcswap dctag dctagkey dctypes dcuid dcwcache dcvm dcvr dcvrae dcvras dcvrat dcvrcs dcvrda dcvrds dcvrdt dcvrfd dcvrfl dcvris dcvrlo dcvrlt dcvrobow dcvrof dcvrpn dcvrpobw dcvrsh dcvrsl dcvrss dcvrst dcvrtm dcvrui dcvrul dcvrulup dcvrus dcvrut dcxfer dcpath vrscan vrscanl)
 BUILD_LIBRARY(dcmdata ${SOURCES})
+find_package( ZLIB REQUIRED )
+if ( ZLIB_FOUND )
+  target_link_libraries( dcmdataShared ${ZLIB_LIBRARIES}  )
+endif( ZLIB_FOUND )
